kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -455,6 +460,18 @@ footer {
     width: 82px;
 }
 
+/* ===================================
+    Modal Dialog Login
+=================================== */
+
+.modal-dialog {
+    width: 400px;
+}
+
+div.panel-body form.form-horizontal .form-group {
```

Comment:
Again, I would rather not do such non-specific selectors there. Can you make this more specific to the login box where we want this behaviour to occur (and hence move it out of bootstrap-overrides also).